databricks/koalas/frame.py

```diff
@@ -22,8 +22,11 @@
 
 import numpy as np
 import pandas as pd
+
 from pyspark import sql as spark
-from pyspark.sql import functions as F
+from collections.abc import Iterable
+from pyspark.sql import Column, DataFrame, functions as F
```

Comment:
We should use `spark.Column` or `spark.DataFrame` instead of importing from `pyspark` since the names conflict with Koalas' ones.
Btw, they are not used?